tests/utils/exec/rosacli/cluster_service.go

```diff
@@ -18,7 +18,7 @@ type ClusterService interface {
 	DescribeCluster(clusterID string) (bytes.Buffer, error)
 	ReflectClusterDescription(result bytes.Buffer) (*ClusterDescription, error)
 	DescribeClusterAndReflect(clusterID string) (*ClusterDescription, error)
-	List() (bytes.Buffer, error)
+	ListCluster(flags ...string) (ListCluster, bytes.Buffer, error)
```

Comment:
Please do not add yet another return type. It is better to have a `ReflectClusterList` like we have for others

Reply:
Hi @radtriste
All other are returning output in form of byte and their responses don't looks like in form of columns, But for `rosa list cluster` it would be efficient to return the data the structure (column wise) format in this way we can easily use the values ( ID, NAME, STATE, etc) easily in automation code.

Reply:
@AkashKanni I was meaning to have a `ReflectClusterList` method similar to `ReflectClusterDescription` which is taking the bugger result and parsing it into struct

Reply:
Can you update following this ?